charts/opentelemetry-collector/examples/old-metrics-url-values/rendered/configmap.yaml

```diff
@@ -87,5 +87,5 @@ data:
           - pull:
               exporter:
                 prometheus:
-                  host: ${env:MY_POD_IP}
+                  host: example.com
```

Comment:
This was the bug. Users that had set a specific value for `metrics.address` were not getting it converted properly.